pkg/apis/upgrade.cattle.io/v1/types.go

```diff
@@ -23,9 +23,13 @@ var (
 )
 
 // +genclient
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Image",type=string,JSONPath=`.spec.upgrade.image`
+// +kubebuilder:printcolumn:name="Channel",type=string,JSONPath=`.spec.channel`
+// +kubebuilder:printcolumn:name="Version",type=string,JSONPath=`.spec.version`
```

Comment:
WDYT about adding a column for current status?

Something like
```
// +kubebuilder:printcolumn:name="Complete",type="string",JSONPath=".status.conditions[?(@.type==\"Complete\")].status"
```

Also couldn't hurt to add `descriptions` here, but I'll let you decide.